cypress/integration/project_spec.js

```diff
@@ -67,6 +67,12 @@ describe('Project', () => {
     })
 
     it('should delete a project with the delete button', () => {
+        for (let i = 0; i < 4; i++) {
```

Comment:
What is happening here? It appears like some kind of hack to make the rest of the test work consistently, but it is not indicated by anything. Maybe a brief comment would be good

Reply:
It creates 4 new projects (arbitrary number) and the purpose of the test is to delete one of them and see that a project got deleted